src/Presentation/RestApi/Controller/v1/TaskRunnerController.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Sdk\Presentation\RestApi\Controller\v1;
+
+use SprykerSdk\Sdk\Core\Application\Service\ContextFactory;
+use SprykerSdk\Sdk\Core\Application\Service\TaskExecutor;
+use SprykerSdk\Sdk\Infrastructure\Mapper\ViolationReportFileMapperInterface;
+use SprykerSdk\SdkContracts\Report\Violation\ViolationReportInterface;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Translation\Exception\InvalidResourceException;
+
+class TaskRunnerController
+{
+    /**
+     * @var \SprykerSdk\Sdk\Core\Application\Service\TaskExecutor
+     */
+    protected TaskExecutor $taskExecutor;
+
+    /**
+     * @var \SprykerSdk\Sdk\Core\Application\Service\ContextFactory
+     */
+    protected ContextFactory $contextFactory;
+
+    /**
+     * @var \SprykerSdk\Sdk\Infrastructure\Mapper\ViolationReportFileMapperInterface
+     */
+    protected ViolationReportFileMapperInterface $violationReportFileMapperInterface;
+
+    /**
+     * @param \SprykerSdk\Sdk\Core\Application\Service\TaskExecutor $taskExecutor
+     * @param \SprykerSdk\Sdk\Core\Application\Service\ContextFactory $contextFactory
+     * @param \SprykerSdk\Sdk\Infrastructure\Mapper\ViolationReportFileMapperInterface $violationReportFileMapperInterface
+     */
+    public function __construct(
+        TaskExecutor $taskExecutor,
+        ContextFactory $contextFactory,
+        ViolationReportFileMapperInterface $violationReportFileMapperInterface
+    ) {
+        $this->taskExecutor = $taskExecutor;
+        $this->contextFactory = $contextFactory;
+        $this->violationReportFileMapperInterface = $violationReportFileMapperInterface;
+    }
+
+    /**
+     * @param string $task
+     *
+     * @throws \Symfony\Component\Translation\Exception\InvalidResourceException
+     *
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    public function __invoke(string $task): Response
+    {
+        $context = $this->contextFactory->getContext();
```

Comment:
Please, do not put any logic inside the controller. Extract it into a separate service instead. Especially if it's an example.

The same for the rest controllers